tutorials/advanced_features/Serialization.ipynb

```diff
@@ -9,15 +9,15 @@
   },
```

Comment:
<p>*as will be seen...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>